docs/src/reference/implementations-tinkergraph.asciidoc

```diff
@@ -198,3 +198,60 @@ g = traversal().withEmbedded(graph)
 g.io("data/tinkerpop-crew.kryo").read().iterate()
 g.V().properties()
 ----
+
+=== Transactions
+
+As of 3.7.0, there is the option to use TinkerTransactionGraph, a transactional TinkerGraph class, to perform traversal
+transactions. This will eliminate the need to use the Neo4j plugin to enable TinkerGraph transactions.
```

Comment:
```suggestion
As of 3.7.0, there is the option to use TinkerTransactionGraph, a transaction-enabled sibling of classic TinkerGraph.
TInkerTransactionGraph is built primarily as a reference implementation of gremlin transactions and for use with TinkerPop transaction tests. It is not intended to be used in production environments.
```

I don't think we should focus too much on Neo4J here, that history will not be relevant after a few releases. Also technically the Neo4J plugin doesn't enable TinkerGraph transactions, it gives a transaction powered alternative to TinkerGraph.

Reply:
I don't think sibling is the right word here. We are trying to avoid the notion that there are too graphs given this comment:

> 3. "In other cases better to use TinkerGraph." - as mentioned earlier, avoid statements that make it sound like there are two "TinkerGraphs". We have one TinkerGraph. If you want to use transactions you use one class and if you don't want transaction support you use a different class. it's still just TinkerGraph.